utils/constants.tsx

```diff
@@ -402,6 +402,7 @@ export const EventTypes = Object.assign(
 );
 
 export const CloudProducts = {
+    STARTER_LEGACY: 'cloud-starter-legacy',
```

Comment:
This will eventually be used once what is currently cloud-starter (and has no limits) becomes the grandfathered cloud-starter plan.